lab02-kaylee/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "lab02-kaylee",
+  "version": "1.0.0",
+  "description": "lab02",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha",
+    "lint": "eslint ./lib",
+    "start": "npm run lint && npm run test"
```

Comment:
**TODO FOR LOST POINTS**
Line 12 should be changed to:

```
"start": "node index.js"
```
This makes it so I can run your ``index`` module by just typing ``npm run start.``